docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,112 @@
+| 71%                   | 0%                    | 0%                    | 100%                  | 100%                  | 99%                | 1%                   |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 50%                   | 0%                    | 0%                    | 50%                   | 0%                    | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 20%                   | 10%                   | 25%                   | 25%                   | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 100%                  | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 72%                   | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 0%                    | 0%                    | 20%                   | 0%                    | 50%                | 50%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+
+Note: The above load balancing uses default :ref:`overprovisioning factor <arch_overview_load_balancing_overprovisioning_factor>` which is 1.4.
+
+To sum this up in pseudo algorithms:
+
+::
+
+  health(P_X) = min(100, 1.4 * 100 * healthy_P_X_backends / total_P_X_backends)
+  normalized_total_health = min(100, Σ(health(P_0)...health(P_X)))
+  priority_load(C_0) = min(100, Σ(health(P_0)...health(P_k)) * 100 / normalized_total_health), where P_0...P_k belong to C_0
+  priority_load(C_X) = min(100 - Σ(priority_load(C_0)..priority_load(C_X-1)),
+                           Σ(health(P_x)...health(P_X)) * 100 / normalized_total_health)
+
+The example shows how the aggregate cluster level load balancer selects the cluster. E.g.,
+healths of {{20, 20, 10}, {25, 25}} would result in a priority load of {70%, 30%} of traffic. When normalized total health drops below 100, traffic is distributed after normalizing
```

Comment:
This seems to omit the fact that we're preserving the relative weighting of priorities within the cluster: it's not just a 70/30 split between C0 and C1, but C0P0 will also get more traffic than C0P1 for example

Reply:
Yeah, the 70/30 is for traffic between clusters. I think we can only know the distribution between clusters but we cannot know the exact traffic distribution among priorities because we don't think the exact load balancing policy the cluster uses.

Reply:
But you're selecting both the cluster and the priority to use when you delegate right? So you're computing the % of traffic that each priority within each cluster is getting

Reply:
Does the % of the traffic to each priority work only for priority-based load balancing algorithm? Or all the load balancing algorithms need to select the priority first.

Reply:
Because of https://github.com/envoyproxy/envoy/pull/7967/files#diff-915049a73fe5779173d4566aba525a57R50 the inner cluster will attempt to select from the specified priority with 100% load given to it, so the priority is completely decided before we delegate lb to the selected cluster

Reply:
Updated the doc